src/stavenote.ts

```diff
@@ -1119,7 +1119,12 @@ export class StaveNote extends StemmableNote {
       ctx.stroke();
     };
 
-    const style = { ...stave.getStyle(), ...(this.getLedgerLineStyle() || {}) };
+    const style = {
+      ...stave.getStyle(),
+      ...(this.getLedgerLineStyle() || {
```

Comment:
Can we update `getLedgerLineStyle` instead?

Reply:
i@0xfe If we do it inside `getLedgerLine` we will have to move exactly the same code in the function to make sure that it defaults to double the "current" `stave.getStyle().lineWidth`, otherwise it will be calculated once and that's it. 
Do you want me to do that?
```
getLedgerLineStyle(): ElementStyle {
    const stave = this.checkStave();
    const style = {
      ...stave.getStyle(),
      ...(this.getLedgerLineStyle() || {
        lineWidth: stave.getStyle().lineWidth * Flow.LEDGER_LINE_THICKNESS_MULTIPLIER,
      }),
    };
    return style;
  }
```

Reply:
Thanks. Actually, I think it's better to initialize `this.ledgerLineStyle` in the constructor to `{ lineWidth: stave.getStyle().lineWidth * 2.0 }` and remove `Flow.LEDGER_LINE_THICKNESS_MULTIPLIER`.

Then this part does not have to change.

Reply:
@0xfe stave is not defined in the constructor. I suggest to initialise simply to 2 as default.

Reply:
Done with simple initialisation to 2.0. Now the changes are really minor.